### PR TITLE
fix: Tests - round 2

### PR DESCRIPTION
**Pull request by KeithClinard**

### CODE REVIEW

File: tests/dummy/app/components/nrg-code-block/template.hbs [hunk 1]
```diff
@@ -1,21 +1,21 @@
-<div class="ui raised segments">
+<div class="ui raised segments code-block">
   <div class="ui segment">
     <h5 class="ui header">
       <i class="icon code"></i>
       <div class="content">
-        {{label}}
+        {{@label}}
         {{#if secondaryLabel}}
```
Comment (TSenter) [inline tests/dummy/app/components/nrg-code-block/template.hbs:7]:
```suggestion
        {{#if @secondaryLabel}}
```

File: tests/dummy/app/view-components/nrg-lightbox/template.hbs [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
   <!-- END-SNIPPET  -->
 </NrgCodeBlock>
 
 <NrgCodeBlock @label="Lightbox Thumbnails with Custom Details" @codeBlockName="lightbox-thumbnails-block.hbs">
   <!-- BEGIN-SNIPPET lightbox-thumbnails-block -->
-  {{#each model as |photo|}}
+  {{#each photos as |photo|}}
     <NrgLightboxThumbnail @photo={{photo}}>
       <strong>{{photo.detail}}</strong>
       <span>{{counter}}</span>
```
Comment (TSenter) [inline tests/dummy/app/view-components/nrg-lightbox/template.hbs:16]:
```suggestion
      <span>{{@counter}}</span>
```

**Review by TSenter — CHANGES_REQUESTED** [review]

File: tests/dummy/app/view-components/nrg-lightbox/template.hbs (hunk 2 above)
Comment (KeithClinard) [reply tests/dummy/app/view-components/nrg-lightbox/template.hbs:16]: This is in a route, and does not need the `@` syntax.

Comment (KeithClinard) [reply tests/dummy/app/view-components/nrg-lightbox/template.hbs:16]: Do you want me to make those changes here, or do you want to make the route changes (`@/this.`) in you PR?

**Review by TSenter — APPROVED** [review]

File: tests/dummy/app/view-components/nrg-lightbox/template.hbs (hunk 2 above)
Comment (TSenter) [reply tests/dummy/app/view-components/nrg-lightbox/template.hbs:16]: I'll take them.

**Pull request merged**